### PR TITLE
Add documentation about how to update the applications

### LINKED ISSUE #17: Document how to modify applications

### PR DESCRIPTION
**Pull request by MichaelKora**
Fixes #17

### CODE REVIEW

File: word-count/README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
 
 Read how to deploy and destroy this pipeline in the official [KPOps Documentation](https://bakdata.github.io/kpops/latest/user/getting-started/quick-start/).
+
+## Modify applications
+
+Create a Pull Request, then after merging, checkout the main branch and run the following commands.
+
+```shell
+# deploy producer app
+gradle jib -Djib.to.image=us.gcr.io/gcp-bakdata-cluster/kpops-word-count-data-producer -Djib.container.mainClass=com.bakdata.kpops.examples.SentenceProducer
```
Comment (philipp94831) [inline word-count/README.md:11]: Please use image names from https://github.com/bakdata/kpops-examples/blob/main/word-count/deployment/kpops/pipeline.yaml

**Review by philipp94831 — CHANGES_REQUESTED** [review]

File: word-count/README.md (hunk 1 above)
Comment (disrupted) [inline word-count/README.md:11]: are the container images available publicly from this registry?

**Review by philipp94831 — APPROVED** [review]

**Review by raminqaf — APPROVED** [review]

File: word-count/README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
 
 Read how to deploy and destroy this pipeline in the official [KPOps Documentation](https://bakdata.github.io/kpops/latest/user/getting-started/quick-start/).
+
+## Modify applications
+
+Create a Pull Request, then after merging, checkout the main branch and run the following commands.
+
+```shell
+# deploy producer app
+gradle jib -Djib.to.image=bakdata/kpops-demo-sentence-producer -Djib.container.mainClass=com.bakdata.kpops.examples.SentenceProducer
```
Comment (disrupted) [inline word-count/README.md:11]: do we need to create those Docker repos first?

Comment (MichaelKora) [reply word-count/README.md:11]: https://hub.docker.com/r/bakdata/kpops-demo-sentence-producer
they already exist

**Review by disrupted — APPROVED** [review]

**Pull request merged**